htdocs/main.inc.php

```diff
@@ -3853,6 +3858,12 @@ function llxFooter($comment = '', $zone = 'private', $disabledoutputofmessages =
 			print $hookmanager->resPrint;
 		}
 
+		$result = $user->call_trigger('AFTER_PAGE_LOAD', $user);
```

Comment:
Triggers are reserved for use inside a transaction and on business CRUD event.
To include code here, hooks are a better choice.
If you miss the hook, you can add it, but it must be a hook, not a trigger.